user_manual/files/access_webdav.rst

```diff
@@ -58,6 +58,1028 @@ You can access files in Linux operating systems using the following methods.
+longer than 30 minutes using Web Client in Windows 7.
+
+Solution
+^^^^^^^^
+
+Workarounds are documented in the KB2668751_ article.
+
+
+Accessing files using cURL
+--------------------------
+
+Since WebDAV is an extension of HTTP cURL can be used to script file operations.
+
+To create a folder with the current date as name:
+
+.. code-block:: bash
+
+	$ curl -u user:pass -X MKCOL "https://example.com/nextcloud/remote.php/dav/files/USERNAME/$(date '+%d-%b-%Y')"
+
+To upload a file ``error.log`` into that directory:
+
+.. code-block:: bash
+
+	$ curl -u user:pass -T error.log "https://example.com/nextcloud/remote.php/dav/files/USERNAME/$(date '+%d-%b-%Y')/error.log"
+
+To move a file:
+
+.. code-block:: bash
+
+	$ curl -u user:pass -X MOVE --header 'Destination: https://example.com/nextcloud/remote.php/dav/files/USERNAME/target.jpg' https://example.com/nextcloud/remote.php/dav/files/USERNAME/source.jpg
+
+To get the properties of files in the root folder:
+
+.. code-block:: bash
+
+	$ curl -X PROPFIND -H "Depth: 1" -u user:pass https://example.com/nextcloud/remote.php/dav/files/USERNAME/ | xml_pp
+	<?xml version="1.0" encoding="utf-8"?>
+    <d:multistatus xmlns:d="DAV:" xmlns:oc="http://nextcloud.org/ns" xmlns:s="http://sabredav.org/ns">
+      <d:response>
+        <d:href>/nextcloud/remote.php/dav/files/USERNAME/</d:href>
+        <d:propstat>
+          <d:prop>
+            <d:getlastmodified>Tue, 13 Oct 2015 17:07:45 GMT</d:getlastmodified>
+            <d:resourcetype>
+              <d:collection/>
+            </d:resourcetype>
+            <d:quota-used-bytes>163</d:quota-used-bytes>
+            <d:quota-available-bytes>11802275840</d:quota-available-bytes>
+            <d:getetag>"561d3a6139d05"</d:getetag>
+          </d:prop>
+          <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+      </d:response>
+      <d:response>
+        <d:href>/nextcloud/remote.php/dav/files/USERNAME/welcome.txt</d:href>
+        <d:propstat>
+          <d:prop>
+            <d:getlastmodified>Tue, 13 Oct 2015 17:07:35 GMT</d:getlastmodified>
+            <d:getcontentlength>163</d:getcontentlength>
+            <d:resourcetype/>
+            <d:getetag>"47465fae667b2d0fee154f5e17d1f0f1"</d:getetag>
+            <d:getcontenttype>text/plain</d:getcontenttype>
+          </d:prop>
+          <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+      </d:response>
+    </d:multistatus>
+
+
+.. _KB2668751: https://support.microsoft.com/kb/2668751
+.. _KB2123563: https://support.microsoft.com/kb/2123563
+.. _WebDAV Navigator: http://seanashton.net/webdav/
+.. _Android devices: https://play.google.com/store/apps/details?id=com.schimera.webdavnavlite
+.. _iPhones: https://itunes.apple.com/app/webdav-navigator/id382551345
+.. _BlackBerry devices: http://appworld.blackberry.com/webstore/content/46816
+======================================
+Accessing Nextcloud files using WebDAV
```

Comment:
Isn't everything below a duplicate of the previous section?

Reply:
There is no mention of blackberries in https://github.com/nextcloud/documentation/blob/vasyugan-patch-1/user_manual/files/access_webdav.rst so I am a bit confused as to where this comes from

Reply:
I think everything below `======================================` is a duplicate. 

![image](https://user-images.githubusercontent.com/3902676/75884924-e6f77780-5e25-11ea-8d5e-fd17e8b2f870.png)

Reply:
What file are you reviewing? This isn't in the file for which I created the pull request.

Reply:
I review this pull request. Just search for Accessing Nextcloud files with your file. It's there two times.

Reply:
super weird. Should be fixed now. Git is confusing. At least for beginners like me.

Reply:
Looks good now. I suggested to revert the text changes I suggested earlier because they are actually not related to this PR.

Reply:
Not sure what I am suppose to do. I have committed the suggested changes, there seems to be no way to un-commit them.